ansible/group_vars/k8s_vm_host/main.yml

```diff
@@ -0,0 +1,4 @@
+k8s_root_path: /home/azure/ubuntu-vm
+k8s_vm_images_url: https://acsbe.blob.core.windows.net/vmimages
+k8s_hdd_image_filename: bionic-server-cloudimg-amd64.qcow2
```

Comment:
>bionic-server-cloudimg-amd64.qcow2 [](start = 24, length = 34)

Is it possible to fetch this image from public URL?
If you did some file format conversion, could you write some code comment about the steps, so in future if the official image upgrades, we could update this image.

Reply:
Yes, I added a comment to describe this